### DISCUSSION TITLE
Guide for Descent Joystick Mapping w/Spaceorb & Orbotron 9001

### DISCUSSION THREAD

**Discussion by gamer85** [thingotron/orb9k_circuitpython#5] [General]
Solution for those looking for SpaceOrb 360 joystick mapping with Descent. With the help of @Yoshimitsu#8541 (Discord), who created an Autocad SpaceOrb360 wireframe diagram, I was able to get the following optimal joystick mapping to work with Descent.
Also see Yoshi's Youtube Channel, esp the video at
www.youtube.com/watch?v=voiENb-WXRU&t=18s

The below are my notes for SpaceOrb260 joystick mapping within Descent 1.  I turned off chording so I would have edge buttons 1 and 2 are trigger buttons for weapons 1 and 2.  Also important to note if there is bleeding of an input direction, please hit the reset button on the back of the SpeceOrb 360 joystick to center inputs.

Fire Primary Button A   (J1 B1)
Fire Secondary Button B (J1 B2)

Button C - Accelerate (J1 B3)
Button E - Reverse    (J1 B5)
Button D - Bank Left  (J1 B6)
Button F - Bank Right (J1 B4)

Axes

Pitch U/D     A6 - RX (roll ball inward)
Turn L/R      A5 - RZ (rotate ball hard left, grab and roll inward)

Slide L/R     A1 - TX (pull ball sideways perpendicularly straight left)
Slide U/D     A3 - TZ (pull ball directly toward you)
Bank L/R      A4 - RY (rotate ball clockwise around Y axis)
Throttle      A2 - TY (pull ball straight up or leave blank)

Note! Because D1 reads the last joystick output, important to hit [ESC] and then use arrow keys then select joystick input then hitting [ENTER] into the desired box.

![Final Joystick Mapping 22-06-19](https://user-images.githubusercontent.com/20948864/174496591-7512619e-00fd-409a-9a38-55abdd492377.jpg)

![SpaceOrb-axes-Descent Vertical](https://user-images.githubusercontent.com/20948864/174496482-39f21892-a0c7-4d62-a251-da4045ece4cc.jpg)
![SpaceOrb-axes-Descent Horizontal](https://user-images.githubusercontent.com/20948864/174496484-a19acd3e-0b7b-4405-915e-d37aa38e4214.jpg)
![SpaceOrb-axes-Descent Least Torque](https://user-images.githubusercontent.com/20948864/174496485-66d2d82a-9287-4147-b69a-76fa8b81a9e7.jpg)

**Comment by vputz** [comment]
This looks fantastic!  Thanks, guys--going to move this to the wiki.

**Reply by shanselman** [reply]
Lovely stuff!
